dlp/snippets/README.rst.in

```diff
@@ -0,0 +1,52 @@
+# This file is used to generate README.rst
```

Comment:
I don't think we need this file anymore. If we want to have a list of samples we should move it to a README.md in one folder above.